core/baseline-core-v1.0-psd01.md

```diff
@@ -4048,60 +4050,229 @@ This section of the document focuses on the requirements and considerations rela
+* The test workgroup is associated with only one test workstep.
+* The test workstep accepts only input data from the agreed-upon data source for a given data schema.
+* The agreed-upon input data source is configured with a test data set.
+* Another, second input data source is configured with the same test data set as the agreed-upon data source. 
+
+Test Steps:
+
+1. Verify that the input data provided to the BPI test workstep is from the agreed-upon authoritative data source.
+2. Verify that the input data complies with the established business logic of the test workstep.
+3. Verify that the BPI workstep processes the input data correctly.
+4. Switch the input data source, and repeat step 1.
+5. Verify that the workstep creates an error, and exits with an appropriate error message.
+
+Test Passing Criteria:
+
+* The BPI workstep processes the first input data correctly.
+* The BPI workstep creates an error, and exits with the expected error message, when the input data comes from the second input data source.
 
 #### **[CR36]>[O10]** 
 If the internal authoritative input data to a BPI workstep is single-sourced, that source MUST be authoritative.
 
-[[CR36]>[O10]](#cr36o10) Testability: An agreement on authoritative data sources will be embedded in a commercial document electronically represented on a BPI as required by [[R6]](#r6) to be bdining. And since [[R6]](#r6) is testable, [[CR36]>[O10]](#cr36o10) is testable.
+[[CR36]>[O10]](#cr36o10) Testability: 
+
+Preconditions:
+
+* A BPI test system is up and running.
+* There is a given set of test BPI Subjects that are part of a test workgroup.
+* The test workgroup is associated with only one test workstep.
+* There exists a system of record identified as the single source for the input data.
+* A test workstep has been created that requires internal authoritative input data from a single source.
+* The BPI Subjects involved in the workstep have agreed that the identified system of record is the authoritative source for the input data.
+* The test workstep accepts only input data from the agreed-upon data source for a given data schema.
+* The agreed-upon input data source is configured with a test data set.
+
+Test Steps:
+
+1. Retrieve the input data for the identified BPI workstep from the single source system of record.
+2. Verify that the single source system of record is the authoritative source for the input data in the test workstep.
+
+Test Passing Criteria:
+
+* The identified single source system of record is confirmed to be the authoritative source for the input data.
 
 ## 8.2 External Authoritative Data for BPIs
 
+###### extauthbpidata
+
 External authoritative data means that the input data to a BPI workstep is held in some authoritative 3rd party database, such as government records. Unlike internal authoritative data where there is only one source, external authoritative data input to a BPI workstep may have multiple sources.
```

Comment:
The statement is that Internal data may come from a single source. This does not mean it does - without an explicit prohibition on coming from multiple sources, or a requirement that it comes from a single source (which would IMHO be silly), there is no difference here.

As a side note, restricting the use of the terms "MUST/SHOULD/MAY" to where they hold RFC2119 meanings, within the text of explicit requirements, probably helps to pick up issues like this and fix them.

Reply:
it should be "...only one version,.,.", not "one source". There may, of course, be multiple versions of the same data yet each from a different source. I will update the wording accordingly. Nice catch! Thank you!